ui_framework/components/menu/_menu.scss

```diff
@@ -1,12 +1,26 @@
+/**
+ * padding-left: 0 and list-style: none rules
+ * are meant for ul.kuiMenu and li.kuiMenuItem
+ * elements.
+ *
+ * kuiMenu and kuiMenuItems can classify both
+ * nonsemantic divs as well as lists (ul, li).
+ */
+.kuiMenu {
+  padding-left: 0;
+}
```

Comment:
Can we structure the comment like this? This will be more consistent with the rest of the comments in the UI Framework.

```scss
/**
 * 1. Allow this class to be applied to `ol` and `ul` elements.
 */
.kuiMenu {
  padding-left: 0; /* 1 */
}
```